bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -205,18 +206,23 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
                 };
             });
 
+            languageCode = mainLanguageCode;
+            if (host.get('request').params.languageCode) {
+                languageCode = host.get('request').params.languageCode;
```

Comment:
The request is not the best place to take the language code. You should get it from `ContentCreateViewService` or `ContentEditViewService` depending if you are in edit or create mode. The current language code is in the attr `languageCode` see: https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/ez-contentcreateviewservice.js#L179